cards/cards.css

```diff
@@ -0,0 +1,90 @@
+@import url(/style.css);
+@import url(/avatar-badge/badge/badge.css);
+
+.card-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.card {
+    background-color: var(--secondary-bg);
+    width: 20rem;
+    border: double 5px var(--secondary);
+    padding: 1rem;
+}
+
+.card-image>img {
+    height: 15rem;
+    width: 20rem;
+}
+
+img {
+    height: 15rem;
+    width: 20rem;
+}
+
+header {
+    font-size: large;
+    font-weight: bolder;
+}
+
+.actions {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.actions p {
+    font-size: 0.9rem;
+    color: var(--primary);
+    padding: 0.3em;
+}
+
+.actions i {
+    padding: 0.3em;
+}
+
+.main-container:hover,
+.actions>p:hover,
+.actions>i:hover,
+.dismiss:hover,
+.actions>.badge-on-icon:hover {
+    background-color: var(--secondary);
+    cursor: pointer;
+}
```

Comment:
just a suggestion: you can make a class hover and then give it the CSS. Now you can use that hover component anywhere, not just in cards

Reply:
Right! Makes sense - will follow it. thanks Maitra